### PR TITLE
[FIX]:Modify the configuration of ODP mode parameters

### LINKED ISSUE #84: [Question]: The ODP pattern should use the ODP parameters alone instead of reusing the original URL
### Description

The ODP pattern should use the ODP parameters alone instead of reusing the original URL,A documentation description should be added

### PR DESCRIPTION
**Pull request by xxsc0529**
## Summary

In ODP mode, you can use ODP-IP and ODP-PORT to connect to the network, and you can leave the URL blank when using ODP mode. close #84 

## Solution Description

Take a look at this unit test.com.oceanbase.connector.flink.OBKVHBaseConnectorITCase

### CODE REVIEW

File: docs/sink/flink-connector-obkv-hbase_cn.md [hunk 1]
```diff
@@ -139,6 +186,28 @@ CREATE TABLE t_sink
...
+) with (
+  'connector'='obkv-hbase',
+  'odp-mode'='true',
+  'odp-ip'='127.0.0.1',
+  'odp-port'='12881',
+  'schema-name'='test',
+  'table-name'='htable1',
+  'username'='root@test',
+  'password'='',
+  'sys.username'='root',
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase_cn.md:207]: Remove sys username and password.

File: flink-connector-oceanbase-base/src/test/java/com/oceanbase/connector/flink/OceanBaseProxyContainer.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public int getSqlPort() {
         return getMappedPort(SQL_PORT);
     }
 
-    public int getRpcPort() {
```
Comment (whhe) [inline flink-connector-oceanbase-base/src/test/java/com/oceanbase/connector/flink/OceanBaseProxyContainer.java:77]: It's better to change the data type at caller side.

**Review by whhe — CHANGES_REQUESTED** [review]

File: docs/sink/flink-connector-obkv-hbase_cn.md (hunk 1 above)
Comment (xxsc0529) [reply docs/sink/flink-connector-obkv-hbase_cn.md:207]: Instructions have been added，sys username and sys password you can leave it blank

File: flink-connector-oceanbase-base/src/test/java/com/oceanbase/connector/flink/OceanBaseProxyContainer.java (hunk 2 above)
Comment (xxsc0529) [reply flink-connector-oceanbase-base/src/test/java/com/oceanbase/connector/flink/OceanBaseProxyContainer.java]: yes, I've modified it

File: docs/sink/flink-connector-obkv-hbase.md [hunk 3]
```diff
@@ -113,6 +115,49 @@ public class Main {
...
+        tEnv.executeSql(
+                "CREATE TABLE t_sink ( "
+                        + " rowkey STRING,"
+                        + " family1 ROW<column1 STRING, column2 STRING>,"
+                        + " PRIMARY KEY (rowkey) NOT ENFORCED"
+                        + ") with ("
+                        + "  'connector'='obkv-hbase',"
+                        + "  'odp-mode'='true',"
+                        + "  'odp-ip'='127.0.0.1',"
+                        + "  'odp-port'='12881',"
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:144]: It's better to take the default rpc '2885' port for example.

File: docs/sink/flink-connector-obkv-hbase_cn.md (hunk 1 above)
Comment (whhe) [reply docs/sink/flink-connector-obkv-hbase_cn.md:207]: It's better to delete it if it's not in use.

File: docs/sink/flink-connector-obkv-hbase_cn.md [hunk 4]
```diff
@@ -152,20 +221,23 @@ VALUES ('1', ROW ('r1f1c1', 'r1f1c2')),
...
+|--------------------------|------|-------|----------|--------------------------------------------------------------------------------------------------|
+| url                      | 否    |       | String   | 集群的 config url，可以通过 <code>SHOW PARAMETERS LIKE 'obconfig_url'</code> 查询。当使用 obconfig_url 模式时为必填项 |
+| schema-name              | 是    |       | String   | OceanBase 的 db 名。                                                                                |
+| table-name               | 是    |       | String   | HBase 表名，注意在 OceanBase 中表名的结构是 <code>hbase_table_name$family_name</code>。                        |
+| username                 | 是    |       | String   | 非 sys 租户的用户名。                                                                                    |
+| password                 | 是    |       | String   | 非 sys 租户的密码。                                                                                     |
+| odp-mode                 | 否    |       | Boolean  | 当 odp-mode 的值为 true 时使用 odp 模式连接 obkv。                                                           |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase_cn.md:231]: There should be a default value.

**Review by whhe — CHANGES_REQUESTED** [review]

File: docs/sink/flink-connector-obkv-hbase_cn.md (hunk 4 above)
Comment (xxsc0529) [reply docs/sink/flink-connector-obkv-hbase_cn.md]: odp-mode add defalut value is false

File: docs/sink/flink-connector-obkv-hbase.md (hunk 3 above)
Comment (xxsc0529) [reply docs/sink/flink-connector-obkv-hbase.md]: updated 2885

File: docs/sink/flink-connector-obkv-hbase.md [hunk 5]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
+| url                      | Yes      |         | String   | The config url, can be queried by <code>SHOW PARAMETERS LIKE 'obconfig_url'</code>.Required when using obconfig_url mode |
+| schema-name              | Yes      |         | String   | The database name of OceanBase.                                                                                          |
+| table-name               | Yes      |         | String   | The table name of HBase, note that the table name in OceanBase is <code>hbase_table_name$family_name</code>.             |
+| username                 | Yes      |         | String   | The username of non-sys tenant.                                                                                          |
+| password                 | Yes      |         | String   | The password of non-sys tenant.                                                                                          |
+| odp-mode                 | No       | false   | Boolean  | If the value of ODP-MODE is true, use the ODP mode to connect to OBKV.                                                   |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:231]:
```suggestion
| odp-mode                 | No       | false   | Boolean  | If set to 'true', the connector will connect to OBKV via ODP.                                                 |
```

File: docs/sink/flink-connector-obkv-hbase.md [hunk 6]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
-| buffer-flush.interval    | No       | 1s      | Duration | Buffer flush interval. Set '0' to disable scheduled flushing.                                                |
-| buffer-flush.buffer-size | No       | 1000    | Integer  | Buffer size.                                                                                                 |
-| max-retries              | No       | 3       | Integer  | Max retry times on failure.                                                                                  |
+|          Option          | Required | Default |   Type   |                                                       Description                                                        |
+|--------------------------|----------|---------|----------|--------------------------------------------------------------------------------------------------------------------------|
+| url                      | Yes      |         | String   | The config url, can be queried by <code>SHOW PARAMETERS LIKE 'obconfig_url'</code>.Required when using obconfig_url mode |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:226]:
```suggestion
| url                      | No      |         | String   | The config url, can be queried by <code>SHOW PARAMETERS LIKE 'obconfig_url'</code>. Required when 'odp-mode' is set to 'false'. |
```

File: docs/sink/flink-connector-obkv-hbase.md [hunk 7]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
+| schema-name              | Yes      |         | String   | The database name of OceanBase.                                                                                          |
+| table-name               | Yes      |         | String   | The table name of HBase, note that the table name in OceanBase is <code>hbase_table_name$family_name</code>.             |
+| username                 | Yes      |         | String   | The username of non-sys tenant.                                                                                          |
+| password                 | Yes      |         | String   | The password of non-sys tenant.                                                                                          |
+| odp-mode                 | No       | false   | Boolean  | If the value of ODP-MODE is true, use the ODP mode to connect to OBKV.                                                   |
+| odp-ip                   | No       |         | String   | IP address of the ODP. When the value of odp-mode is true, it is required                                                |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:232]:
```suggestion
| odp-ip                   | No       |         | String   | IP address of the ODP. Required if 'odp-mode' is set to 'true'.        |
```

File: docs/sink/flink-connector-obkv-hbase.md [hunk 8]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
+| odp-port                 | No       |         | Integer  | rpc_listen_port of ODP. When the value of odp-mode is true, it is required.                                              |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:233]:
```suggestion
| odp-port                 | No       |         | Integer  | rpc_listen_port of ODP. Required if 'odp-mode' is set to 'true'.                                             |
```

File: docs/sink/flink-connector-obkv-hbase.md [hunk 9]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
+| sys.username             | No       |         | String   | The username of sys tenant.Required when using obconfig_url mode                                                         |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:234]:
```suggestion
| sys.username             | No       |         | String   | The username of sys tenant. Required if 'odp-mode' is set to 'false'.                                                          |
```

File: docs/sink/flink-connector-obkv-hbase.md [hunk 10]
```diff
@@ -152,20 +221,23 @@ Once executed, the records should have been written to OceanBase.
...
+| sys.password             | No       |         | String   | The password of sys tenant.Required when using obconfig_url mode                                                         |
```
Comment (whhe) [inline docs/sink/flink-connector-obkv-hbase.md:235]:
```suggestion
| sys.password             | No       |         | String   | The password of sys tenant. Required if 'odp-mode' is set to 'false'.                                                        |
```

**Review by whhe — COMMENTED** [review]
I suggest to modify the options order, and move 'odp-mode' up as the first optional parameter. For the text part, it's better to add a section to describe about `What is odp-mode`, or `What does 'odp-mode'=true' means`.

**Review by whhe — APPROVED** [review]
LGTM

**Pull request merged**